test/unit/ui/hash.test.js

```diff
@@ -4,8 +4,8 @@ import window from '../../../src/util/window';
 import { createMap as globalCreateMap } from '../../util';
 
 test('hash', (t) => {
-    function createHash() {
-        const hash = new Hash();
+    function createHash(name) {
```

Comment:
Could you add an additional test for `Hash`, so that there is a test with and without a `hashName` parameter.

Reply:
I actually _added_ a named version for most of the tests, so it should be enough IMHO. See lines `createHash` usages on lines 22, 34, 47, 77, 125, 173 for unnamed and 105, 143, 219 for named.

That or I did not understand your comment.

Reply:
My mistake @SebCorbin , I misread this function in my previous read.